blockly-rtc/src/index.js

```diff
@@ -43,7 +43,7 @@ document.addEventListener('DOMContentLoaded', () => {
         if (event instanceof Blockly.Events.Ui) {
             return;
         };
-        workspaceClient.addEvent(event.toJson());
+        workspaceClient.activeChanges.push(event.toJson());
```

Comment:
Why did you get rid of `addEvent` and have the workspace client expose its `activeChanges` array instead?

Reply:
I created `addEvent` to take in an event, create an object of the form `{event: event, entryId: entryIdString}` and add that object to `notSent`.
Now, that we are not creating an object, just adding an event to `activeChanges` I removed it. However, I do like the idea of keeping the function `addEvent` (just changing its functionality) as to not expose the `activeChanges` array.
What do you think?

Reply:
I think it's a good idea to keep `addEvent` to avoid exposing the extra array.